.github/workflows/release.yml

```
@@ -79,6 +79,19 @@ jobs:
         with:
           validate-wrappers: true
 
+      - name: Install Syft (SBOM generation tool)
+        shell: bash
+        run: |
+          set -euo pipefail
+          echo "Installing Syft for binary SBOM generation..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)